src/erlcloud_ecs.erl

```diff
@@ -2591,6 +2591,7 @@ to_binary(undefined) -> undefined;
 to_binary(true) -> true;
 to_binary(false) -> false;
 to_binary(L) when is_list(L), is_list(hd(L)) -> [to_binary(V) || V <- L];
+to_binary(L) when is_list(L), is_binary(hd(L)) -> [to_binary(V) || V <- L];
```

Comment:
why not just  `....-> L;`

Reply:
I'd like to make sure that all elements in the list are binary, not just the head one.